selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/refactor/packages/RenamePackageTest.java

```diff
@@ -698,7 +700,16 @@ private void waitTextInNewNameField(String expectedText) {
       refactor.waitTextIntoNewNameField(expectedText);
     } catch (TimeoutException ex) {
       // remove try-catch block after issue has been resolved
-      fail("Known issue https://github.com/eclipse/che/issues/7500");
+      fail("Known issue https://github.com/eclipse/che/issues/7500", ex);
+    }
+  }
+
+  private void clickOkButtonIntoRefactorForm() {
```

Comment:
The method **clickOkButtonIntoRefactorForm()** looks like specific for page object _Refactor_.
Due to _Single Responsibility Principle_ it's better to move _try-catch_ block inside the method of dedicated PageObject **Refactor::clickOkButtonIntoRefactorForm()**.

Reply:
At the same time, if **clickOkButtonIntoRefactorForm()** method is failing because of it's disabled and IDE displays server error message in the form, it's more reasonable to check on absence of server error message and fail if the message is present:
![screenshot from 2017-11-28 18-20-11](https://user-images.githubusercontent.com/1197777/33331148-80ab332e-d469-11e7-82bb-86f18e7ff41a.png)